tabular/src/autogluon/tabular/trainer/abstract_trainer.py

```diff
@@ -212,8 +212,7 @@ def train(self, X_train, y_train, X_val=None, y_val=None, **kwargs):
     # TODO: Remove name_suffix, hacked in
     # TODO: Enable easier re-mapping of trained models -> hyperparameters input (They don't share a key since name can change)
     def train_multi_levels(self, X_train, y_train, X_val, y_val, X_unlabeled=None, hyperparameters: dict = None, base_model_names: List[str] = None,
-                           hyperparameter_tune=False, feature_prune=False, core_kwargs: dict = None, level_start=0, level_end=0,
-                           stack_name_core='core', stack_name_aux='aux1', time_limit=None, name_suffix='') -> List[str]:
+                           hyperparameter_tune=False, feature_prune=False, core_kwargs: dict = None, aux_kwargs: dict = None, level_start=0, level_end=0, time_limit=None, name_suffix='') -> List[str]:
         """
         Trains a multi-layer stack ensemble using the input data on the hyperparameters dict input.
```

Comment:
docstring should clarify how `core_kwargs, aux_kwargs` are passed to core/aux models.

Reply:
will write docstrings in a future PR, currently a lot of functionality is being updated.